frontend/integration-tests/tests/login.scenario.ts

```diff
@@ -0,0 +1,49 @@
+import { $, browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '../protractor.conf';
+import * as loginView from '../views/login.view';
+
+const JASMINE_DEFAULT_TIMEOUT_INTERVAL = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+const JASMINE_EXTENDED_TIMEOUT_INTERVAL = 1000 * 60 * 3;
+
+describe('Auth test', () => {
+  beforeAll(async() => {
+    await browser.get(appHost);
+  });
+
+  const {BRIDGE_AUTH_USERNAME, BRIDGE_AUTH_PASSWORD} = process.env;
```

Comment:
We might make this `KUBEADMIN_PASSWORD` if we expect this to be the kubeadmin user always (in which case we don't need `BRIDGE_AUTH_USERNAME`)